content/documentation/overview/index.adoc

```diff
@@ -0,0 +1,123 @@
+---
+title: "Overview"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Overview Contents
```

Comment:
I think it should always  be "Content", not in plural

Reply:
Thanks, I need help to this document, I don't know exactly if you wanna a resume of all the Application or a full description of each page and some review of the writing will be great maybe Julie can help here.